docs/_guides/codewind-eclipse-quick-guide.md

```diff
@@ -92,10 +89,11 @@ Your application is now opened in a browser, showing the welcome to your Appsody
 ### Adding a REST service to your application
 
 1. Go to your project's workspace under the Project Explorer tab.
-2. Navigate to `Java Resources->src/main/java->dev.appsody.starter`.
+2. Go to `Java Resources->src/main/java->dev.appsody.starter`.
```

Comment:
Format of arrows should match style guide. https://github.com/eclipse/codewind-docs/wiki/Writing-style-guidelines#arrows
Code phrase format might be appropriate here, though, because these are folder/file names.